core/base/src/TApplication.cxx

```diff
@@ -1067,8 +1067,9 @@ void TApplication::Help(const char *line)
       Printf("   .qqqqqqq            : abort process");
       Printf("   .which [file]       : shows path of macro file");
       Printf("   .![OS_command]      : executes OS-specific shell command");
+      Printf("   .!root -?           : print ROOT usage (CLI options)");
       Printf("\nROOT terminal keyboard shortcuts.");
-      Printf("* This command might be intercepted by MacOS predefined system shortcuts."); // https://apple.stackexchange.com/questions/18043/how-can-i-make-ctrlright-left-arrow-stop-changing-desktops-in-lion
+      Printf("* This command might be intercepted by macOS predefined system shortcuts."); // https://apple.stackexchange.com/questions/18043/how-can-i-make-ctrlright-left-arrow-stop-changing-desktops-in-lion
```

Comment:
In `.help`?! Please keep this short, without documentation on "how do I configure my system". Just the info that this might not be available on macOS is enough, and it'd be nice if that's only shown in macOS.

Reply:
I have changed this part so that it only shows up if it's a Mac and no documentation on how to configure the system.

Reply:
and changed "Mac" to "mac"